scripts/transpile/tasks/replaceStylableImports.js

```diff
@@ -2,7 +2,7 @@ const fs = require('fs');
 const glob = require('glob');
 const path = require('path');
 
-const targetDir = path.resolve(__dirname, '..', 'dist/es/src');
+const targetDir = path.resolve(__dirname, '../../../', 'dist/es/src');
```

Comment:
this is meh comment, but for the purists, there should be no '/' in `path.resolve`. It will add slashes for you. This way it will also work well on windows

```suggestion
const targetDir = path.resolve(__dirname, '..', '..', '..', 'dist', 'es', 'src');
```